### PR TITLE
Fhicls for LE production

### PR DESCRIPTION
**Pull request by lpaulucc**
This PR introduces new fhicl files or modify existing ones for the LE production. Changes are related to
- New decay0 radiological model for VD (v3.5) and HD (v3.6)
- Adding simulation for half the active volume in HD to emulate lateral APA
This PR goes together with https://github.com/DUNE/dunecore/pull/107 and https://github.com/DUNE/dunesim/pull/66

(STILL UNDER DEVELOPMENT)

### CODE REVIEW

File: fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_1x2x6.fcl:17]: Could you reindent and drop the whitespace?

File: fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl:6]: did this fcl not simulate backgrounds before?

File: fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_1x2x6.fcl:17]: Can you reindent and fix the whitespace?

File: fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl:6]: This fcl did not simulate backgrounds before?

File: fcl/dunefd/g4/supernova_g4_halfActiveVol_dune10kt_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/g4/supernova_g4_halfActiveVol_dune10kt_1x2x6.fcl:1]: Could this fcl inherit from the supernova fcl?  Then only a small number of overrides need to be implemented here

File: fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl
Comment (absolution1) [inline fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl:76]: Did anything change here?

File: fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl
Comment (absolution1) [inline fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl:178]: Just to double check, we are just dropping v3_5?

File: fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl
Comment (absolution1) [inline fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl:273]: whitespace and reindent

File: fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_1x2x6.fcl
Comment (absolution1) [inline fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_1x2x6.fcl:11]: good catch :)

**Review by absolution1 — APPROVED** [review]
Excellent stuff, thank you.
I've posted cosmetic requests-only and had one question about v3_5 to v3_6.  Those shouldn't hold this up so I am approving now :)

File: fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl
Comment (lpaulucc) [reply fcl/dunefd/gen/supernova/prodmarley_nue_cc_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl:6]: This fhicl is new, actually... But I did have to add this line after the first commit to change from the low activity that it inherited.

File: fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl
Comment (lpaulucc) [reply fcl/dunefd/gen/supernova/prodmarley_nue_es_flat_radiological_decay0_dune10kt_halfActiveVol_1x2x6.fcl:6]: Same thing as the other comment.

File: fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl
Comment (lpaulucc) [reply fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl:76]: The 1x2x2 workflow was not setup in the file.

File: fcl/dunefd/g4/supernova_g4_halfActiveVol_dune10kt_1x2x6.fcl
Comment (lpaulucc) [reply fcl/dunefd/g4/supernova_g4_halfActiveVol_dune10kt_1x2x6.fcl:1]: Absolutely :-)

File: fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl
Comment (lpaulucc) [reply fcl/dunefd/gen/background/workflow_radiological_decay0_dune10kt.fcl:178]: HD v3.5 was prepared and then soon changed to v3.6. So I don't see why we should keep it around in the configs (I am only keeping v3.4 around because we will eventually come back to it, once the APA geometry is fixed)

**Pull request merged**
